.github/CHANGELOG.md

```diff
@@ -1,61 +1,115 @@
-# Release 0.17.0-dev (development release)
+# Release 0.16.0 (current release)
```

Comment:
```suggestion
# Release 0.17.0-dev (development release)

<h3>New features since last release</h3>

* Ising YY gate functionality added.
  [(#1358)](https://github.com/PennyLaneAI/pennylane/pull/1358)

<h3>Improvements</h3>

<h3>Breaking changes</h3>

<h3>Bug fixes</h3>

<h3>Documentation</h3>

<h3>Contributors</h3>

This release contains contributions from (in alphabetical order):

Ashish Panigrahi


# Release 0.16.0 (current release)
```

Reply:
We don't want to lose the changes from master :)

Reply:
Oops, great catch! Thank you!